src/cli/commands/files/add.js

```diff
@@ -182,7 +188,8 @@ module.exports = {
       strategy: argv.trickle ? 'trickle' : 'balanced',
       shardSplitThreshold: argv.enableShardingExperiment ? argv.shardSplitThreshold : Infinity,
       'cid-version': argv['cid-version'],
-      'raw-leaves': argv['raw-leaves']
+      'raw-leaves': argv['raw-leaves'],
+      onlyHash: argv['only-hash']
```

Comment:
it should be `'only-hash'`

Reply:
Saw you just pushed a change for this, thanks. You'll need to change all/most instances of `onlyHash`, including in http-api/resources/files.js. I converted it all to onlyHash as dot properties are easier to work with and read. Interestingly, `yargs` also automatically converts all hyphenated options to camelCase: this could also be `onlyhash: argv.onlyHash`

Reply:
Actually, I suggest we keep it `onlyHash`. `cid-version` and `raw-leaves` are the outlier here. There are many hyphenated options we convert to camelCase at the entry points of cli/http-api such as the above `shardSplittingThreshold` and `dag --local-resolve`, `object --input-enc`.

I can change raw-leaves and cid-version instead

Reply:
@JonKrone I like that. Go for it :)